File: lib/parameters.py
from __future__ import print_function

def parse_mle(file_path):
    store = {}
    file = open(file_path, 'r')
    lines = file.readlines()
    for line in lines:
        kws = line.replace('e76','eps76').replace('e81','eps81').split(' ')
        if (kws[0] == 'parameterSummary') & kws[2].startswith('{mle='):
            v = float(kws[2].replace('{mle=','').replace(';',''))
            store[kws[1]] = v
    return store

File: lib/parameters.jl
function parseSummary(fname)
    file = open(fname)
    lines = readlines(file)
    p = Dict{String,Float64}()
    for line in lines
        els = split(line, " ")
        if els[1] == "parameterSummary"
            k = els[2]
            if startswith(k, "exrep1.") == false
                str = strip(SubString(els[3], 6), ';')
                if str != ""
                    v = parse(Float64, str)
                    p[k] = v
                end
            end
        end
    end
    return p
end
